dusty/systems/docker/testing_image.py

```diff
@@ -5,6 +5,12 @@
 from ...compiler.compose import lib_install_commands_for_app_or_lib, container_code_path, get_volume_mounts
 from ...log import log_to_client
 
+def _remove_image_if_exists(docker_client, image_name):
+    try:
+        docker_client.remove_image(image=image_name)
+    except:
+        log_to_client('was not able to remove image {}'.format(image_name))
```

Comment:
had it inline initially. but needed to do it two places.   if i do it with a function, i am not littering try excpets multiple places